plugin/inc/ticket.class.php

```diff
@@ -417,7 +417,8 @@ public function showForm($ID, $options = []): bool
     {
         $this->initForm($ID, $options);
         $this->setPrinter($options['printerId'] ?? null);
-        $location = $this->getLocation();
+        $partnerId = $this->printer->fields['supplier_id'] ?? $options['partnerId'] ?? ($ID > 0 ? $this->getFirstAssignedPartner()->getID() : null);
```

Comment:
If the ticket already has a partner assigned, it should not be overwritable
```suggestion
        $partnerId = ($ID > 0 ? $this->getFirstAssignedPartner()->getID() : null) ?? $this->printer->fields['supplier_id'] ?? $options['partnerId'] ?? null;
```